### DISCUSSION TITLE
How can use tables with filter

### DISCUSSION THREAD

**Discussion by marco1730417** [cdmoro/bootstrap-vue-3#754] [General]
Please help me. I couldn't implement the table with filter .

**Comment by VividLemon** [comment]
Hypothetically you could implement your own filter... the documentation ain't great and I don't exactly know how it works either. One day it will all work right lmao

**Reply by VividLemon** [reply]
https://github.com/cdmoro/bootstrap-vue-3/issues/740#issuecomment-1286244031
@devhus

**Comment by devhus** [comment]
Hello, You can filter the table internally as mentioned in the full example in this PR https://github.com/cdmoro/bootstrap-vue-3/pull/643 where you use these props ``:filter`` as the filter file(ex: search string), and ``filterable`` as an array of filterable field keys.

If you need to filter the items through server-side queries, you should use the ``provider`` prop as mentioned in this PR https://github.com/cdmoro/bootstrap-vue-3/pull/678
